source/specify-documents-to-return.txt

```diff
@@ -0,0 +1,179 @@
+.. _java-rs-specify-documents-to-return:
+
+===========================
+Specify Documents to Return
+===========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, paginate, pagination, order, code example
+
+Overview
+--------
+
+In this guide, you can learn how to specify which documents to return from a read
+operation by using the following methods:
+
+- ``limit()``: Specifies the maximum number of documents to return from a query.
+- ``sort()``: Specifies the sort order for the returned documents.
+- ``skip()``: Specifies the number of documents to skip before returning query results.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:ref:`<java-rs-getting-started>`.
+
+.. include:: includes/reactor-note.rst
+
+Limit
+-----
+
+To specify the maximum number of documents returned from a read operation,
+chain the ``limit()`` method to a ``find()`` method and pass the number
+of documents you want to return.
+
+The following example finds all restaurants that have a ``cuisine`` field value
+of ``"Italian"``, and limits the results to 5 documents:
+
+.. io-code-block::
+
+   .. input:: /includes/specify-documents-to-return.java
+      :start-after: start-limit-method
+      :end-before: end-limit-method
+      :language: java
+
+   .. output::
+
+      Isle Of Capri Resturant Italian
+      Arturo'S Italian
+      Patsy'S Italian Restaurant Italian
+      Piccola Venezia Italian
+      Roadhouse Restaurant Italian
+
+.. tip::
+
+   The preceding example returns the first five documents returned by the query, in
+   no particular order. The following section describes how to return the documents
+   in a specified sort order.
+
+.. _java-rs-sort:
+
+Sort
+----
+
+To return documents in a specified order, chain the ``sort()`` method to a
+``find()`` method. The ``sort()`` method takes one parameter: a sort direction
+method. To specify the sort direction, pass either the ``ascending()`` or ``descending``
```

Comment:
[suggestion] Potentially reword this to avoid using a [colon](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#colons) in a sentence: 
```suggestion
``find()`` method. The ``sort()`` method takes the sort direction
method as a parameter. To specify the sort direction, pass either the ``ascending()`` or ``descending``
```